server/server.go

```diff
@@ -1388,6 +1400,12 @@ func (s *Server) v1DataPost(w http.ResponseWriter, r *http.Request) {
 			rego.StrictBuiltinErrors(strictBuiltinErrors),
 		}
 
+		for _, r := range s.manager.GetWasmResolvers() {
```

Comment:
When a new bundle downloads and activates do these get invalidated by the server's reload function?

Reply:
I assume we're doing this for performance? A short comment explaining why this is done would be useful.

Reply:
> When a new bundle downloads and activates do these get invalidated by the server's reload function?

Yep, the prepared rego objects get invalidated, and the manager will update its set of them as well. So the next time we do an eval we will have to create a rego object, which will get the updated ones.

> I assume we're doing this for performance? A short comment explaining why this is done would be useful.

Sort of, as-is the Rego object, when preparing for eval, will look in the store for any resolvers it needs to instantiate unless some were explicitly provided with options. So it does affect performance because it would inadvertently initialize a bunch of wasm vm's when doing the prepare call, which it didn't need to do as we have all of them already initialized on the manager. TL;DR: I'll add a comment 😄